modules/linear_boltzmann_solvers/lbs_solver/lbs_solver.cc

```diff
@@ -442,39 +432,38 @@ LBSSolver::OptionsBlock()
 
   params.SetGeneralDescription("Set options from a large list of parameters");
   params.SetDocGroup("LBSUtilities");
-
   params.AddOptionalParameter("spatial_discretization",
                               "pwld",
                               "What spatial discretization to use. Currently only `\"pwld\"` "
                               "is supported");
   params.AddOptionalParameter(
     "scattering_order", 1, "Defines the level of harmonic expansion for the scattering source.");
   params.AddOptionalParameter("max_mpi_message_size",
-                              32'768,
+                              32768,
                               "The maximum MPI message size used during sweep initialization.");
   params.AddOptionalParameter(
     "read_restart_data", false, "Flag indicating whether restart data is to be read.");
+  params.AddOptionalParameter("read_restart_directory_name",
+                              opensn::input_path.stem().string() + "_restart",
+                              "Directory name to use when reading restart data.");
+  params.AddOptionalParameter("read_restart_file_stem",
+                              opensn::input_path.stem().string(),
+                              "File stem to use when reading restart data.");
```

Comment:
This could really all be encapsulated in a single attribute called `read_restart_dump` which could be defaulted to an empty string (no restart read), or specified as `<dir>/<stem>`.